server/schedule/operator/builder.go

```diff
@@ -243,6 +244,28 @@ func (b *Builder) SetLeader(storeID uint64) *Builder {
 	return b
 }
 
+// SetLeaders records all valid target leaders in Builder.
+func (b *Builder) SetLeaders(storeIDs []uint64) *Builder {
+	if b.err != nil {
+		return b
+	}
+	for _, storeID := range storeIDs {
+		if peer, ok := b.targetPeers[storeID]; !ok {
+			continue
+		} else if core.IsLearner(peer) {
```

Comment:
```suggestion
		if core.IsLearner(peer) {
```
can avoid use `else`.

Reply:
Refactored, but not completely in the suggested way.